src/helm/benchmark/presentation/run_specs_chat_gpt.conf

```diff
@@ -96,35 +96,37 @@ entries: [
   # like `author` and `plantiff`.
-  {description: "wikifact:model=openai/chat-gpt,k=5,subject=plaintiff", priority: 2}
-  {description: "wikifact:model=openai/chat-gpt,k=5,subject=place_of_birth", priority: 2}
-  {description: "wikifact:model=openai/chat-gpt,k=5,subject=medical_condition_treated", priority: 2}
-  {description: "wikifact:model=openai/chat-gpt,k=5,subject=instance_of", priority: 2}
-  {description: "wikifact:model=openai/chat-gpt,k=5,subject=part_of", priority: 2}
-  {description: "wikifact:model=openai/chat-gpt,k=5,subject=currency", priority: 2}
-  {description: "wikifact:model=openai/chat-gpt,k=5,subject=position_held", priority: 2}
-  {description: "wikifact:model=openai/chat-gpt,k=5,subject=author", priority: 2}
-  {description: "wikifact:model=openai/chat-gpt,k=5,subject=discoverer_or_inventor", priority: 2}
-  {description: "wikifact:model=openai/chat-gpt,k=5,subject=symptoms_and_signs", priority: 2}
+  # Skip for now since it's more expensive to run (num_outputs=5)
+  # {description: "wikifact:model=openai/chat-gpt,k=5,subject=plaintiff", priority: 2}
+  # {description: "wikifact:model=openai/chat-gpt,k=5,subject=place_of_birth", priority: 2}
+  # {description: "wikifact:model=openai/chat-gpt,k=5,subject=medical_condition_treated", priority: 2}
+  # {description: "wikifact:model=openai/chat-gpt,k=5,subject=instance_of", priority: 2}
+  # {description: "wikifact:model=openai/chat-gpt,k=5,subject=part_of", priority: 2}
+  # {description: "wikifact:model=openai/chat-gpt,k=5,subject=currency", priority: 2}
+  # {description: "wikifact:model=openai/chat-gpt,k=5,subject=position_held", priority: 2}
+  # {description: "wikifact:model=openai/chat-gpt,k=5,subject=author", priority: 2}
+  # {description: "wikifact:model=openai/chat-gpt,k=5,subject=discoverer_or_inventor", priority: 2}
+  # {description: "wikifact:model=openai/chat-gpt,k=5,subject=symptoms_and_signs", priority: 2}
 
 
   ##### Reasoning #####
 
   # Got response: "I'm sorry, but I cannot solve the problem as you have not provided
   # enough information. You have only provided the rules and target for each part of
   # the problem, but not the actual values for X, Y, and Z. Without this information"
-  {description: "synthetic_reasoning:model=openai/chat-gpt,mode=pattern_match", priority: 2}
-  {description: "synthetic_reasoning:model=openai/chat-gpt,mode=variable_substitution", priority: 2}
-  {description: "synthetic_reasoning:model=openai/chat-gpt,mode=induction", priority: 2}
+  # {description: "synthetic_reasoning:model=openai/chat-gpt,mode=pattern_match", priority: 2}
+  # {description: "synthetic_reasoning:model=openai/chat-gpt,mode=variable_substitution", priority: 2}
+  # {description: "synthetic_reasoning:model=openai/chat-gpt,mode=induction", priority: 2}
 
+  # ChatGPT tends to responds "Nothing can be determined about X".
   {description: "synthetic_reasoning_natural:model=openai/chat-gpt,difficulty=easy", priority: 2}
```

Comment:
should this be commented?

Reply:
Should we include it to highlight that ChatGPT sometimes responds this way?

Reply:
Ya but also when it doesnt (to indicate toxicity isnt solved). I.e. contrast between textdavinci003 and chatgpt will also be interesting